docs/user/supported_devices.rst

```diff
@@ -414,6 +414,10 @@ Footnotes
   The device or target is reaching its end of life soon. This means that support
   in the next major release of Gluon is doubtful.
 
+.. [#featureset-tiny]
+  These devices do not support the full featureset of Gluon. Namely, WPA3 support
+  for the private WLAN is missing on these devices.
```

Comment:
Needs better explanation; should refer to the part of `site.rst` where setting the features for the featuresets is described.

Reply:
We also need to document the criteria for setting the featuresets of devices somewhere.

As this PR is about adding WPA3 and the size of the OpenSSL libraries, I'd expect "tiny" to be about the flash size only, but somehow in the target definitions below you also set `tiny` for 8/32?